### PR TITLE
updating caption - raj

### PR DESCRIPTION
**Pull request by rajanchd**
closes, fixes.

### CODE REVIEW

**Comment by rajanchd** [comment]
Test

File: _slides/01-intro.md [hunk 1]
```diff
@@ -2,5 +2,5 @@
 layout: slide
 title: "Welcome to our slide deck!"
 ---
-
+author: "Rajan Rawat"
```
Comment (tstanekovic94) [inline _slides/01-intro.md:5]: Looks good

**Review by tstanekovic94 — APPROVED** [review]
approved

**Comment by rajanchd** [comment]
dpme

**Pull request merged**